discord/ext/commands/default.py

```diff
@@ -70,12 +70,14 @@ async def default(self, ctx, param):
         raise NotImplementedError('Derived classes need to implement this.')
 
 
-class Author(CustomDefault):
+class CurrentAuthor(CustomDefault):
     """Default parameter which returns the author for this context."""
 
     async def default(self, ctx, param):
         return ctx.author
 
+Author = CurrentAuthor
```

Comment:
Since this isn't merged into the lib yet, probably best not to introduce extra aliases. Anyone depending on this code in it's pre-submitted form should be prepared for it to break (e.g., I'll update my bot in case we do want this rename)

Moved discussion about the actual rename to https://github.com/Rapptz/discord.py/pull/1849#issuecomment-822045842

Reply:
Well, I can't actually talk in there.